src/main/kotlin/org/onflow/flow/sdk/crypto/Crypto.kt

```diff
@@ -156,11 +159,31 @@ object Crypto {
 }
 
 internal class HasherImpl(
-    private val hashAlgo: HashAlgorithm
+    private val hashAlgo: HashAlgorithm,
+    private val key: ByteArray? = null
```

Comment:
@tarakby , yes the ? indicates a nullable argument so that the function could be called with either 1 or 2 arguments. Will have a look at the other 2 arguments for customiser and output length to add them in as well - thanks!

Reply:
Yeah, from chatting to Tarak to securely integrate BLS into the SDK would be a big effort, so we're going to not add that. The likely audience for that addition is very small and they would be ok to use Go SDK instead should they need the narrow use cases which BLS is associated with